src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/UpdateNuGetConfigPackageSourcesMappings.cs

```diff
@@ -34,48 +40,90 @@ public class UpdateNuGetConfigPackageSourcesMappings : Task
         /// </summary>
         public string[] SourceBuildSources { get; set; }
 
+        [Required]
+        public string SbrpRepoSrcPath { get; set; }
+
+        private const string SbrpCacheSourceName = "source-build-reference-package-cache";
```

Comment:
It would be ideal if this could be passed in as a task property so it's shared with the value from here: https://github.com/dotnet/installer/blob/79f35dcddc1a4c48fdd46e55b9563c5d9f4cba6e/src/SourceBuild/content/repo-projects/source-build-reference-packages.proj#L30